src/plugins/intel_gpu/src/graph/include/layout_optimizer.h

```diff
@@ -188,6 +188,9 @@ class layout_optimizer {
 
     impl_types get_forced_impl_type_by_config(program_node& node);
     static bool are_data_types_suitable_for_onednn(program_node& node);
+    static bool check_data_types_for_pooling(data_types src, data_types dst);
+    static bool check_data_types_for_convolution(data_types src, data_types weight, data_types dst);
+    static bool check_data_types_for_fc_gemm(data_types src, data_types weight, data_types dst);
```

Comment:
If it is to be used only from layout_optimizer.cpp, what about defining it as static function within cpp file?

Reply:
They will be removed.